app/rtc_s1.py

```diff
@@ -147,6 +148,122 @@ def apply_slc_corrections(burst_in: Sentinel1BurstSlc,
     del band_out
 
 
+def calculate_layover_shadow_mask(burst_in: Sentinel1BurstSlc,
+                                 geogrid_in: isce3.product.GeoGridParameters,
+                                 path_dem: str,
+                                 filename_out: str,
+                                 threshold_rdr2geo: float=1.0e-7,
+                                 numiter_rdr2geo: int=25,
+                                 extraiter_rdr2geo: int=10,
+                                 lines_per_block_rdr2geo: int=1000,
+                                 threshold_geo2rdr: float=1.0e-8,
+                                 numiter_geo2rdr: int=25,
+                                 dem_block_margin_geo2rdr: float=0.1,
```

Comment:
Please, remove this argument

Reply:
Do you mean all of those 4 parameters (`lines_per_block_rdr2geo`, `threshold_geo2rdr`, `numiter_geo2rdr`, and `dem_block_margin_geo2rdr`) or the one parameter `dem_block_margin_geo2rdr` that we do not use?

Reply:
Oh sorry, only `dem_block_margin_geo2rdr`.

Reply:
Thanks Gustavo. The change has pushed to the fork. :)